docker-compose.yaml

```diff
@@ -52,13 +56,57 @@ services:
       PHX_HOST: ${BACKEND_ADDRESS:-$DOMAIN}
       PORT: 5004
     restart: unless-stopped
+    depends_on:
+      - jellyfish
     ports:
       - "5004:5004"
+    networks:
+      - jellyfish_network
+
+  prometheus:
+    build:
+      context: ./prometheus
+      dockerfile: Dockerfile
+    container_name: prometheus
+    restart: on-failure
+    command:
+      - --config.file=/etc/prometheus/prometheus.yml
+      - --storage.tsdb.path=/prometheus
+      - '--web.console.libraries=/usr/share/prometheus/console_libraries'
+      - '--web.console.templates=/usr/share/prometheus/consoles'
+    volumes:
+      - prometheus_data:/prometheus
     depends_on:
       - jellyfish
+    ports:
+      - 9090:9090
     networks:
       - jellyfish_network
 
+  grafana:
+    build:
+      context: ./grafana
+      dockerfile: Dockerfile
+    container_name: grafana
+    restart: on-failure
+    volumes:
+      - grafana_data:/var/lib/grafana
+    depends_on:
+      - prometheus
+    environment:
+      GF_SECURITY_ADMIN_PASSWORD: "${GF_SECURITY_ADMIN_PASSWORD}"
+      GF_SECURITY_ADMIN_USER: "${GF_SECURITY_ADMIN_USER}"
+    ports:
+      - '3000:3000'
+    expose:
+      - 3000
+    networks:
+      - jellyfish_network
```

Comment:
Not really connected to this PR, but `docker compose` creates "bridge" network and connects all of the defined services to it, so I believe that everything would work even without all of the network configuration (maybe better be explicit tho?).

Reply:
You are right I thought that network configuration was needed, so I will remove it.